Backend/src/main/java/DiceRoller.java

```diff
@@ -0,0 +1,48 @@
+package main.java;
+
+import java.util.Random;
+
+/**
+ * A local dice roller which generates a random number output based on the
+ * requested dice type
+ */
+
+public class DiceRoller {
+
+    private Random dice;
+
+    /**
+     * Initialize a DiceRoller object based on the random library
+     */
+    public DiceRoller() {
+        dice = new Random();
+    }
+
+    /**
+     * This function represents a dice roll. The theoretical range of the number
+     * of sided dice is infinite, so long as it is positive
+     * 
+     * @param diceCount The number of times we want a specific dice to be rolled
+     * @param modifier  The modifer that gets tagged at the end of a "dice roll"
+     * @param DiceType  The type of dice being rolled
+     * 
+     * @return Randomly generated number based on diceCount + the modifier + dice
+     *         type
+     */
+    int Roll(int diceCount, int modifier, int DiceType) {
```

Comment:
Shouldn't DiceType be the enum you made?

Reply:
then you can do `dice.nextInt(DiceType.range)`

Reply:
DiceType.range is an integer, the function doesn't take an ENUM, it takes an integer. When the roll function is called, it'll be like this:
dice.Roll(1, 0, Dice.D20.range)

Reply:
I don't remember what it was that stopped me from specifically just passing in the ENUM. I might have done it wrong.

Reply:
Ye, I was questioning not having the variable be an Enum, this would ensure no bogus inputs being passed ex: (DiceType = 10000, DiceType = -100000)

Reply:
Done.